src/plugins/intel_gna/gna_graph_compiler.cpp

```diff
@@ -882,6 +882,18 @@ void GNAGraphCompiler::PoolingPrimitive(InferenceEngine::CNNLayerPtr layer) {
     uint32_t num_padding = ALIGN(hw_in, 8) - hw_in;
     size_t num_data_bytes_in = c_dim_in * (hw_in + num_padding) * inputs->getPrecision().size();
 
+    if (dnn->new_num_conv_columns) {
```

Comment:
I understand that everybody knows what new_num_conv_columns  mean, but could you please add the short comment, personally for me :)

Reply:
In the line 379 you can find few comments about new_num_conv_columns and an algorithm that calculates it. In general, @elilobanova, please, correct me if I'm wrong, this variable is used to store an output of convolution taking into account padding.

Reply:
Line 379 is a part of another function and it is not clear here why we are processing it (dnn->new_num_conv_columns) here. Could you please comment such logic next time